client_interface.go

```diff
@@ -9,7 +9,7 @@ type ClientInterface interface {
 	// Exported methods
 	CheckSubscription(listID string, email string) (*MemberResponse, error)
 	Subscribe(listID string, email string, mergeFields map[string]interface{}) (*MemberResponse, error)
-	UpdateSubscription(listID string, email string, status string, mergeFields map[string]interface{}) (*MemberResponse, error)
+	UpdateSubscription(listID, email string, params map[string]interface{}) (*MemberResponse, error)
```

Comment:
You didn't remove `string` from `listID string` in the other method signatures. I'd either remove them all or leave this one for consistency.

Reply:
Done! Added it back.